HACKING.md

```diff
@@ -153,22 +153,20 @@ To do that, you will need to download the debuginfo artifact that matches the Ap
 
 ### AppImage-Debugging using manual symbol loading
 
-For using this approach you need to first unpack the debug information:
+For using this approach you need to first unpack the debug information
+downloaded from the release:
 
 ```bash
-$ unzip debuginfo.zip
-Archive:  debuginfo.zip
-  inflating: hotspot-debuginfo-v1.4.0-43-g02fd82c-x86_64.tar.bz2
 $ tar -xvf hotspot-debuginfo-*.tar.bz2
-hotspot-debuginfo-v1.4.0-43-g02fd82c/usr/bin/hotspot
-hotspot-debuginfo-v1.4.0-43-g02fd82c/usr/lib64/libexec/hotspot-perfparser
+hotspot-debuginfo-v1.5.0-17-g0ed41cf/usr/bin/hotspot
```

Comment:
here and below. unrelated changes?

Reply:
We could make it possibly even more obvious: leaving the sha as is, but changing the version to 1.2.3.-45 - Do you agree?
Are you fine with changing the date in the example below or should I leave this as-is?